src/components/content/credential/AddCredential.tsx

```diff
@@ -56,8 +56,20 @@ function AddCredential({
     }, [credentialTypesQuery.data, credentialTypesQuery.isSuccess]);
 
     useEffect(() => {
-        setCredentialTypeList([]);
-    }, [credentialTypesQuery.error]);
+        if (currentCsp !== undefined) {
+            const error = credentialTypesQuery.error as Error;
+            if (error instanceof ApiError && 'details' in error.body) {
+                const response: Response = error.body as Response;
+                getTipInfo('error', response.details.join());
+                // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+            } else if (error !== undefined && error !== null) {
```

Comment:
this warning is valid. We don't need the error undefined or null checks. we can have the default else block here and remove the  line 64.